tests/components/application-connector/test/compass-runtime-agent/config.go

```diff
@@ -0,0 +1,16 @@
+package compass_runtime_agent
+
+import "fmt"
+
+type config struct {
+	DirectorURL                  string `envconfig:"default=http://compass-director.compass-system.svc.cluster.local:3000/graphql"`
+	SkipDirectorCertVerification bool   `envconfig:"default=false"`
+	OauthCredentialsNamespace    string `envconfig:"default=test"`
+	OauthCredentialsSecretName   string `envconfig:"default=oauth-compass-credentials"`
+	TestingTenant                string `envconfig:"default=tenant"`
+}
```

Comment:
The env variables with the Compass data are specified in [this comment](https://github.com/kyma-project/test-infra/pull/5977#issuecomment-1240414698). Do they somehow correspond to this config or these are two separate things?

Reply:
All needed env variables will be provided via in secret named  `OauthCredentialsSecretName`